### ISSUE TITLE
GamePass(insiderUpdate)

### ISSUE THREAD

**Issue by WonderingViper** [SutandoTsukai181/RyuModManager#4]
Hello. I'm fairly new to these things and am at a lost on as of how to fix this error. I'm playing the GamePass version but Yakuza 0 and Kiwami 1 have worked flawlessly with your manager. Kiwami 2 does not however work, so the only method for me to Mod it is manually placing and replacing (using ParTool/Parc-Archive-Importer) downloaded mods from Nexus. Is there any possible way you could fix this? and BTW xbox app (insider) on PC now allows you to download games wherever you choose and allows file editing so that is big. The manual method I'm using is time consuming and sometimes a hit and miss so although not being able to use RMM for Kiwami 2 is a small inconvenience, It still would be awesome to have things automated with your handy tool again. Thanks for taking the time if any to read this.

![Kiwami2-Assertion Failed](https://user-images.githubusercontent.com/94955537/143198927-1c4f15f8-a1d9-4565-9f8d-42a28c591eba.png)
![Kiwami2-BreakpointTriggered](https://user-images.githubusercontent.com/94955537/143198929-22484ca9-522d-4f51-bea3-d9dfdb527eab.png)
![Kiwami2-ExceptionThrown](https://user-images.githubusercontent.com/94955537/143198931-3d0931fa-eab7-4295-96a3-5b719e55ff30.png)
![Kiwami2-ExceptionUnhandled](https://user-images.githubusercontent.com/94955537/143198934-11f5b7f8-cede-45ce-9e82-5626937fc31d.png)
![Kiwami2-Files](https://user-images.githubusercontent.com/94955537/143198937-bb540ecc-8170-4668-a2c7-e34d7154478c.png)

**Comment by mosamadeeb** [comment]
Hey there. Does the mod manager work fine with Yakuza 0 and Kiwami 1 with the latest xbox app update? i.e. did you have to dump the games, or are you directly accessing the game files (without dumping)?

As for Kiwami 2, there is a possibility that the game's executable is different from the one in the latest steam version, so that might be causing this issue. Only way to fix this is to add support by finding some specific instructions in the UWP version of the game, which I don't own currently. I won't be able to fix this right now, but I'll keep it in mind for later.

**Comment by WonderingViper** [comment]
Yes both games worked perfectly with the new GamePass (Insider) update, you can edit files directly no UWP dumping needed. And okay that sounds promising, I tried to fix it myself having no knowledge of coding or languages. After a few dozen debugging and coding videos I realized I bit off more than I could chew. Sorry for the late reply I was exhausted.

**Comment by WonderingViper** [comment]
I'm still going to try and fix it on my end, It's just going to take me time to learn these things and apply them. Do you have any suggestions on where to begin? I honestly want to learn these things so I could help in the future with projects such as these.

**Comment by AbdolAhmed** [comment]
Did you find a fix for this yet?

**Comment by ImANormalUser** [comment]
anything yet?

**Comment by jfmherokiller** [comment]
"As for Kiwami 2, there is a possibility that the game's executable is different from the one in the latest steam version," 

I would say get the sha1 hashes of both exes and compare (if anyone is still reading this)

**Comment by AhnJaeHoon-0512** [comment]
Problems continue to arise even now.

**Comment by steakysudo** [comment]
Has this been fixed?
